webapp/pom.xml

```diff
@@ -70,21 +70,13 @@
                 <artifactId>domain</artifactId>
                 <version>1.0-SNAPSHOT</version>
             </dependency>
-            <dependency>
-                <groupId>com.vaadin</groupId>
-                <artifactId>vaadin-bom</artifactId>
-                <version>${vaadin.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
```

Comment:
why did you move the vaadin dependency? I would expect it on the app layer and not in the webapp since only here there should be UI stuff.

Reply:
true, i have refactored it